docs/exceptions.rst

```diff
@@ -0,0 +1,55 @@
+Runtime Exceptions
+==================
+
```

Comment:
We should document what it means 'can be thrown by the graphql engine'. Does it mean I get an exception in `GraphQL.execute' or a special error etc ...

Reply:
updated